force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -71,8 +82,16 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Positive", "Negative", or "Zero", or null if the number is null.
      */
     public static String checkNumber(Integer a) {
+        if(a == null){
         return null; // Replace null with the variable you used to store the result
-    }
+        } else if (a > 0) {
+            return 'Positive';
+        } else if (a == 0){
+            return 'Zero';
+        } else {
+            return 'Negative';
+        }
+        }
```

Comment:
Careful of the indenting here. It is much easier for readability when the indenting is vertically the same by bracket.